clients/admin-ui/cypress/e2e/systems.cy.ts

```diff
@@ -231,141 +169,6 @@ describe.skip("System management page", () => {
           cy.url().should("match", /system$/);
         });
       });
-
-      it("Can render and post extended form fields", () => {
```

Comment:
this got moved to the editing related tests, since now you can only fill out the extended fields when editing